qm-plugins/qm-vip/class-qm-vip-output-html.php

```diff
@@ -26,7 +26,7 @@ public function output() {
 			<?php
 			if ( isset( $data['mu-plugins']['commit'] ) && isset( $data['mu-plugins']['date'] ) ) {
 				echo '<p><a href="https://github.com/automattic/vip-go-mu-plugins/commit/' . rawurlencode( $data['mu-plugins']['commit'] ) .
-				'" alt="GitHub URL of the commit that the stack was deployed from."><i><strong>Last modified: </strong>' . esc_html( $data['mu-plugins']['date'] ) .
+				'" alt="GitHub URL of the commit that the stack was deployed from." target="_blank"><i><strong>Last modified: </strong>' . esc_html( $data['mu-plugins']['date'] ) .
```

Comment:
`alt` attribute isn't valid on `a` elements - I think it meant `title`.

And the `title` attribute isn't accessibility friendly, and shouldn't be used here.

And if the link is being forced into a new tab, then it needs a screen-reader-text span to indicate to screen reader users that it opens in a new tab.